lec12-datasets.Rmd

```diff
@@ -0,0 +1,159 @@
+    - Learn basics of group dynamics in a team setting, how to share responsibilities and duties
+    - Get into groups and start your projects
+
+### Lesson outline:
+
+Total lesson time: 2 hours
+
+- Datasets available for use (10-20 min)
+- What is the scientific method (20 min)
+- Team dynamics and assigning roles and tasks (15-20 min)
+- Break (10 min)
+- Getting into groups (10 min)
+    - Exercise (25-30 min)
+    - Start project work (20-25 min)
+
+-----
+
+## Datasets for use
+
+### List of datasets available
+
+{{list/link them here}}
+
+## Scientific method
+
+<!-- What is the scientific method? -->
+
+### Simplified diagram of the scientific method
+
+```{r fig_scientific_method, echo=FALSE}
+DiagrammeR::grViz('
+digraph rmarkdown {
+    graph [layout = neato, splines = true]
+    node [shape = box, style = rounded]
+    Hypothesis [pos = "0.75,3.75!"]
+    CollectData [label = "Collect Data", pos = "0.75,3!"]
+    AnalyzeData [label = "Analyze Data", pos = "0.75,2.25!"]
+    HypothesisTrue [label = "Hypothesis\nif TRUE", pos = "0,1.5!"]
+    HypothesisFalse [label = "Hypothesis\nif FALSE", pos = "1.5,1.5!"]
+    Report [label = "Report/Publish", pos = "0.75,0.75!"]
+
+    Hypothesis -> CollectData -> AnalyzeData -> HypothesisTrue -> Report
+    AnalyzeData -> HypothesisFalse -> {Report Hypothesis}
+}')
+```
+
+### Steps in the scientific process[^sci_method]
+
+1. Identify research question(s).
+2. Look into what the previous literature shows. <!--(how much emphasis on this?)-->
+3. Create one or more hypotheses or objectives.
+4. Write up an outline or expected approach to answering those questions/objectives (analysis and presentation plan):
+    - How will the data be obtained and what is the data (i.e. the source).
+    - What statistical or mathematical techniques have previous researchers used? Will they be used or will other ones be used? Are they appropriate (optional, may need an expert)? 
+    - How will the results/data be presented or visualized?
+    - Identify/describe some possible tables and figures to present the results/data.
+5. Run all analyses (or additional ones that may come up as you work with and analyze the data).
+6. Visualize or present all results from the analyses.
+7. Interpret the results and how they fit in the context of the previous literature and findings.
+8. Draw conclusions based on the hypotheses/objectives.
+9. Publish your results (in blogs, pre-print archives, general archiving servers, journals, or at conferences)
+
+
+[^sci_method]: See [Khan Academy on Scientific Method](https://www.khanacademy.org/science/biology/intro-to-biology/science-of-biology/a/the-science-of-biology) for a brief overview and explanation on the scientific process.
+
+## Team dynamics
+
+### Basics of succeeding with a team
+
+Final assignment is in a group and to succeed, you need to understand team dynamics:
+
+- Everyone is different, every team is different
```

Comment:
I think the 'everyone is unique' message is not that helpful in this context. There are sets of guiding principles that will benefits most teams (including those we are trying to teach in this class). Sure, it is good to be open to that the problems you encounter will depend *in part* on your team's composition and you should be flexible in how to solve them, although also here there are certainly already established approaches of doings so.